community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -147,7 +147,7 @@ public StateHandlingStatementOperations(
     @Override
     public Cursor<NodeItem> nodeCursor( KernelStatement statement, long nodeId )
     {
-        Cursor<NodeItem> cursor = statement.getStoreStatement().acquireSingleNodeCursor( nodeId );
+        Cursor<NodeItem> cursor = statement.getStoreStatement().acquireSingleNodeCursor( nodeId, statement );
```

Comment:
That would be interesting yeah. I'd expect a very small (unnoticable) impact since the volatile `open` flag was checked once per property request (one or more properties) previously, now it's once per property.